test/nested.jl

```diff
@@ -50,16 +58,22 @@
   @test γ(Point(10.,0.), Point(0.,0.)) ≈ sill(γ)
   @test isstationary(γ)
 
-  # test constructor explicitly
+  # test coe-6tructor explicitly
```

Comment:
```suggestion
  # test constructor explicitly
```

Reply:
Misuse of automatic replace, sorry for that ..